ipmi-config/ipmi-config-category-core-lan-conf-section.c

```diff
@@ -36,6 +37,7 @@
 #include "freeipmi-portability.h"
 #include "pstdout.h"
 
+#define BMC_MAXIPV6ADDRLEN 40
```

Comment:
nit-pick cleanup, could IPv6 macro be after the IPv4 macro, since I think we're trying that order everywhere.